### PR TITLE
Added dropdowns and updated dropdown css from maya-ui

### PR DESCRIPTION
**Pull request by Pallavi-PH**
https://www.figma.com/file/Wx0Cl31QWdMvRYHrtBXqE5/Decode?node-id=103%3A2

### CODE REVIEW

File: scss/product/components/_dropdown.scss [hunk 1]
```diff
@@ -9,13 +9,19 @@
   @include caret;
 }
 
+//three dot menu toggle for drop down
```
Comment (visualapps) [inline scss/product/components/_dropdown.scss]: where we use this?

File: scss/product/components/_dropdown.scss [hunk 2]
```diff
@@ -9,13 +9,19 @@
...
+}
+
 // The dropdown menu
 .dropdown-menu {
   position: absolute;
   top: 100%;
   left: 0;
   z-index: $zindex-dropdown;
-  display: none; // none by default, but block on "open" of the menu
+  // display: none; // none by default, but block on "open" of the menu
```
Comment (visualapps) [inline scss/product/components/_dropdown.scss]: display will be none for menu, by default.

to show the preview in <Playground> we can overwrite display with utils class 

eg
`<div class="dropdown-menu d-block">
`

File: scss/product/components/_dropdown.scss (hunk 1 above)
Comment (Pallavi-PH) [reply scss/product/components/_dropdown.scss]: We are not using this. I have removed the unused css

File: scss/product/components/_dropdown.scss (hunk 2 above)
Comment (Pallavi-PH) [reply scss/product/components/_dropdown.scss]: If we add util class to display in preview, It would also be visible in the code block.

Comment (Pallavi-PH) [reply scss/product/components/_dropdown.scss]: Added a separate class '`decode`' for displaying outside `playground` in style.scss

**Pull request merged**